<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>1920</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="option2Button">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>750</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/new/prefix/ASSETS/kombuchaSP.png</normaloff>:/new/prefix/ASSETS/kombuchaSP.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>400</width>
      <height>400</height>
     </size>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="option1Button">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>300</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/new/prefix/ASSETS/coldbrewJJ.png</normaloff>:/new/prefix/ASSETS/coldbrewJJ.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>400</width>
      <height>400</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="option3Button">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>1200</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/new/prefix/ASSETS/elderflower.png</normaloff>:/new/prefix/ASSETS/elderflower.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>400</width>
      <height>400</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="option1Label">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>300</y>
      <width>400</width>
      <height>350</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75pt &quot;Ubuntu&quot;;
background-color: rgba(0, 0, 0,0);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:72pt; color:#2e3436;&quot;&gt;Label&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="option2Label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>750</y>
      <width>400</width>
      <height>350</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75pt &quot;Ubuntu&quot;;
</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:72pt; color:#2e3436;&quot;&gt;Label&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="option3Label">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>1200</y>
      <width>400</width>
      <height>350</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75pt &quot;Ubuntu&quot;;
</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:72pt; color:#000000;&quot;&gt;Label&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="option1InfoButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>620</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 57 italic 30pt &quot;Ubuntu&quot;;
</string>
    </property>
    <property name="text">
     <string>?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="option2InfoButton">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>1070</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 57 italic 30pt &quot;Ubuntu&quot;;
</string>
    </property>
    <property name="text">
     <string>?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="option3InfoButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>1520</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 57 italic 30pt &quot;Ubuntu&quot;;
</string>
    </property>
    <property name="text">
     <string>?</string>
    </property>
   </widget>
   <widget class="QLabel" name="option3SoldOutLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>1200</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:84pt; font-weight:600; color:#444444; vertical-align:super;&quot;&gt;Sold Out&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="option1SoldOutLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>300</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:84pt; font-weight:600; color:#444444; vertical-align:super;&quot;&gt;Sold Out&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="option2SoldOutLabel">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>750</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:84pt; font-weight:600; color:#444444; vertical-align:super;&quot;&gt;Sold Out&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="surveyButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>1700</y>
      <width>550</width>
      <height>100</height>
     </rect>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/new/prefix/ASSETS/elderflower.png</normaloff>:/new/prefix/ASSETS/elderflower.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>400</width>
      <height>400</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="option3ALabel">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>1450</y>
      <width>400</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75pt &quot;Ubuntu&quot;;
</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:72pt; color:#000000;&quot;&gt;Label&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="option2ALabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>990</y>
      <width>400</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75pt &quot;Ubuntu&quot;;
</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:72pt; color:#000000;&quot;&gt;Label&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="option1ALabel">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>550</y>
      <width>400</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75pt &quot;Ubuntu&quot;;
</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:72pt; color:#000000;&quot;&gt;Label&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="logoLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>700</width>
      <height>180</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
